include/librealsense2/dds/topics/notification/notification-msg.h

```diff
@@ -4,46 +4,57 @@
 #pragma once
 #include <string>
 #include <librealsense2/h/rs_sensor.h>
-#include "notificationsPubSubTypes.h"
+#include "notificationPubSubTypes.h"
 
 namespace librealsense {
 namespace dds {
 namespace topics {
 namespace device {
-class notifications
+class notification
 {
 public:
+    using type = raw::device::notificationPubSubType;
 
-    using type = raw::device::notificationsPubSubType;
+    // Currently we use constant MAX size of profiles,
+    // If we decide we want a scaled solution we may need to split the profiles to 
+    // message for each profile (But then we have multiple transport overhead..)
+    static const size_t MAX_VIDEO_PROFILES = 200;
+    static const size_t MAX_MOTION_PROFILES = 20;
 
 #pragma pack( push, 1 )
     enum class msg_type : uint16_t
     {
         DEVICE_HEADER,
-        STREAM_HEADER,
+        SENSOR_HEADER,
         VIDEO_STREAM_PROFILES,
         MOTION_STREAM_PROFILES,
         MAX_MSG_TYPE
     };
 
-    struct device_header
+    struct device_header_msg
     {
-        size_t num_of_streams;
+        size_t num_of_sensors;
     };
 
-    struct stream_header
+    struct sensor_header_msg
     {
-        uint8_t index;           // Index of the current stream [0-(num_of_streams-1)]
-        char name[32];           // Stream name
-        size_t num_of_profiles;  // Profile count for this stream
+        enum sensor_type // maybe enum class with 1 byte inheritance 
+        {
+            VIDEO_SENSOR,
+            MOTION_SENSOR
+        };
+
+        sensor_type type;
+        uint8_t index;           // Index of the current sensor [0-(num_of_sensors-1)]
+        char name[32];           // Sensor name
     };
 
 
     struct video_stream_profile
     {
-        int8_t index;           // Sensor index (Normally )
+        int8_t index;           // LRS Sensor index
```

Comment:
If it's the sensor index, can we call it `sensor_index`?

Reply:
(and maybe add that it's the same index as in `sensor_header_msg`)

Reply:
Why is this index even here, anyway? We have the index in the _msg...

Reply:
It is not the same index.
I agree it's confusing 
We have index --> `lrs_sensor_index` (used by LibRS to distinguish IR L/R) - I will rename it as you suggested
And `dds_sensor_index` (Just to index the DDS sensor currently being sent/received [0 - num_of_sensors) )

On each profile we have `lrs_sensor_index`, on the _msg we have `dds_sensor_index`

Reply:
Wait, My mistake.
The proper name in `LibRS` is `stream_index` , I'll rename to that one.